[docs] Adds Vexo to monitoring services (#34981)
expo/expo: docs/pages/guides/using-vexo.mdx

## Features

1. **Complete Dashboard**
   - Active Users
   - Session Time
   - Downloads
   - OS Distribution
   - Version Adoption
   - Geographic Insights
   - Popular Screens
2. **Session Replays**
   - Watch real user sessions to understand their interactions.
3. **Heatmaps**
   - Identify the most engaged areas of your app.
4. **Funnels**
   - Analyze user flows and optimize conversion rates.
5. **Custom Events and Dashboard Personalization**
   - Track specific user actions by creating custom events.
   - Customize your dashboard to visualize key metrics.

## Getting started

1. Create an account: Sign up for a [Vexo account](https://www.vexo.co/).
2. Create a new app: You'll be prompted to create a new app. Give it a name (you can change it later), and once you submit it, you'll receive an API key.
3. Install the Vexo package: Run the following command in your project:

   <Tabs>
     <Tab label="npm">
       <Terminal cmd={['$ npm install vexo-analytics']} />
     </Tab>
     <Tab label="yarn">
       <Terminal cmd={['$ yarn add vexo-analytics']} />
     </Tab>
   </Tabs>

4. Initialize Vexo: Add the following code in your app's entry file (for example, **index.js**,

>    **App.js**, or **\_layout.tsx** if using Expo Router):

Reviewer: ```suggestion
   **App.js**, or **app/\_layout.tsx** if using Expo Router):
```

Let's mention the `app/` prefix to indicate that it has to be used in the Root layout file.

```tsx app/_layout.tsx
import { vexo } from 'vexo-analytics';

// You may want to wrap this with `if (!__DEV__) { ... }` to only run Vexo in production.
vexo('YOUR_API_KEY');
```

5. Rebuild and run your app: Since `vexo-analytics` includes native code, you need to rebuild your application.

6. Verify integration: Go to your app's page on Vexo and you should see your first event!

## Compatibility

> - Expo: Vexo is compatible with [Development Builds](https://docs.expo.dev/development/introduction/) and does not require additional configuration plugins.

Reviewer: ```suggestion
- Expo: Vexo is compatible with [Development builds](/development/introduction/) and does not require additional configuration plugins.
```

- Expo Go: Not supported, as Vexo requires custom native code.

## Learn more about Vexo

> To learn more about how to use Vexo with Expo, check out the [Vexo documentation](https://docs.vexo.co/).

Reviewer: ```suggestion
To learn more about using Vexo with Expo, check out the [Vexo documentation](https://docs.vexo.co/).
```
